src/components/tooltip/Tooltip.tsx

```diff
@@ -1,21 +1,7 @@
 import * as React from 'react';
-import { Popover, Text } from '@/index';
-import { PopoverProps } from '@/index.type';
+import { Popover, Text, PositionType } from '@/index';
 import { BaseProps } from '@/utils/types';
 
-// const tooltipPropsList = [
-//   'trigger',
-//   'on',
-//   'open',
-//   'offset',
-//   'onToggle',
-//   'dark',
-//   'customStyle',
-//   'closeOnBackdropClick',
-//   'hideOnReferenceEscape',
-//   'closeOnScroll',
-// ] as const;
-// type TooltipPopperProps = typeof tooltipPropsList[number];
 export interface TooltipProps extends BaseProps {
   /**
    * Text to be rendered in `Tooltip`
```

Comment:
Can we use some other prop name instead of `tooltip`? "tooltip" as a prop name inside the "tooltip" component doesn't look good.